bpf/tests/skip_lb_xlate_lrp_per_packet_lb.c

```diff
@@ -0,0 +1,333 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+/* Copyright Authors of Cilium */
+
+#include "common.h"
+#include <bpf/ctx/unspec.h>
+#include <bpf/api.h>
+#include "pktgen.h"
+
+#define ENABLE_IPV4 1
+#define ENABLE_IPV6 1
+#undef ENABLE_HEALTH_CHECK
+#define ENABLE_LOCAL_REDIRECT_POLICY 1
+#define ENABLE_SOCKET_LB_HOST_ONLY 1
+#define HAVE_NETNS_COOKIE 1
+
+/* Set a dummy value for netns cookie*/
+#ifdef ENDPOINT_NETNS_COOKIE
+    #undef ENDPOINT_NETNS_COOKIE
+#endif
+#define ENDPOINT_NETNS_COOKIE 5000
+
+#include <bpf_lxc.c>
+
+#define FROM_CONTAINER 0
+
+struct {
+	__uint(type, BPF_MAP_TYPE_PROG_ARRAY);
+	__uint(key_size, sizeof(__u32));
+	__uint(max_entries, 2);
+	__array(values, int());
+} entry_call_map __section(".maps") = {
+	.values = {
+		[FROM_CONTAINER] = &cil_from_container,
+	},
+};
+
+#define V4_SERVICE_IP		v4_svc_one
+#define SERVICE_PORT		tcp_svc_one
+#define V4_BACKEND_IP		v4_pod_one
+#define BACKEND_PORT		__bpf_htons(8080)
+#define V6_SERVICE_IP		v6_pod_one
+#define V6_BACKEND_IP		v6_pod_two
+
+PKTGEN("tc", "v4_local_redirect")
+int  v4_local_backend_to_service_packetgen(struct __ctx_buff *ctx)
+{
+	struct pktgen builder;
+	volatile const __u8 *src = mac_one;
+	volatile const __u8 *dst = mac_two;
+	struct ethhdr *l2;
+	struct iphdr *l3;
+	struct tcphdr *l4;
+	void *data;
+	/* Init packet builder */
+	pktgen__init(&builder, ctx);
+	/* Push ethernet header */
+	l2 = pktgen__push_ethhdr(&builder);
+	if (!l2)
+		return TEST_ERROR;
+	ethhdr__set_macs(l2, (__u8 *)src, (__u8 *)dst);
+	/* Push IPv4 header */
+	l3 = pktgen__push_default_iphdr(&builder);
+	if (!l3)
+		return TEST_ERROR;
+	l3->saddr = V4_BACKEND_IP;
+	l3->daddr = V4_SERVICE_IP;
+	/* Push TCP header */
+	l4 = pktgen__push_default_tcphdr(&builder);
+	if (!l4)
+		return TEST_ERROR;
+	l4->source = tcp_src_one;
+	l4->dest = SERVICE_PORT;
```

Comment:
Use `pktgen__push_ipv4_tcp_packet()` here? (same in the other PKTGEN stage below)

Reply:
https://github.com/cilium/cilium/pull/34443.